com.unity.cinemachine/Runtime/Core/CinemachineDebug.cs

```diff
@@ -31,6 +35,62 @@ public static Rect GetScreenPos(Camera camera, string text, GUIStyle style)
         }
 #endif
 
+        
+#if UNITY_EDITOR
+        const string k_DebugUIName = "CinemachineDebugUI";
+        static GameObject s_UIDocumentHolder;
+        static UIDocument s_UIDocument;
+        public static VisualElement CreateRuntimeUIContainer()
+        {
+            if (s_UIDocumentHolder == null)
+            {
+                s_UIDocumentHolder = GameObject.Find(k_DebugUIName);
+                if (s_UIDocumentHolder == null)
+                {
+                    s_UIDocumentHolder = new GameObject(k_DebugUIName)
+                    {
+                        hideFlags = HideFlags.NotEditable | HideFlags.HideInHierarchy,
+                        tag = "EditorOnly"
+                    };
+                }
+            }
+            if (!s_UIDocumentHolder.TryGetComponent(out s_UIDocument))
+            {
+                s_UIDocument = s_UIDocumentHolder.AddComponent<UIDocument>();
+                
+                const string path = "Packages/com.unity.cinemachine/Runtime/UI/";
+                s_UIDocument.panelSettings = AssetDatabase.LoadAssetAtPath<PanelSettings>(path + "CinemachinePanelSettings.asset");
```

Comment:
This line is the only thing that's preventing the CinemachineDebug service to be usable at runtime - which could be an interesting thing to have.  Can you think of a way to build this asset in-memory?

Reply:
I'll make a separate task for making it work in build too. I talked to UITK team a bit and it is not a clear.

"It is really preferable you create it in the editor as we are using the panel settings to include some assets in builds, like the theme and the shaders"
So we would need to create a persistent debug ui gameobject that's in all scenes hidden in case user wants to debug in build.

They said there may be a generic build debug console in the future ("not soon") that we could use instead.